docs/content/en/docs/tasks/packages/prereq.md

```diff
@@ -0,0 +1,33 @@
+## Prerequisites
+Before installing any curated packages for EKS Anywhere, do the following:
+
+* Check that the version of `eksctl anywhere` is `v0.9.0` or above with the following commands:
```

Comment:
```suggestion
* Check that the version of `eksctl anywhere` is `v0.11.0` or above with the following commands:
```

Reply:
This whole thing is a bit of a toubleshooting doc. I like the include of this partial, but wonder if we should just link to troubleshooting given one or two checks here

Reply:
Well, I guess it doesn't have to look like a troubleshooting doc. The idea was to let someone know that they must have some things in place before they just try to install a package. By putting it in one file, we can just read it into each packages task doc.